drivers/usbdev_synopsys_dwc2/usbdev_synopsys_dwc2.c

```diff
@@ -578,15 +580,20 @@ static void _sleep_periph(const dwc2_usb_otg_fshs_config_t *conf)
     /* Unblocking STM32_PM_STOP during suspend on the stm32f446 breaks
      * while (un)blocking works on the stm32f401, needs more
      * investigation with a larger set of chips */
-#ifdef STM32_USB_OTG_CID_1x
+#if defined(STM32_USB_OTG_CID_1x)
     pm_unblock(STM32_PM_STOP);
+#elif defined(MCU_EFM32)
+    pm_block(0);
+    pm_unblock(1);
 #endif
 }
 
 static void _wake_periph(const dwc2_usb_otg_fshs_config_t *conf)
 {
-#ifdef STM32_USB_OTG_CID_1x
+#if defined(STM32_USB_OTG_CID_1x)
     pm_block(STM32_PM_STOP);
+#elif defined(MCU_EFM32)
+    pm_block(1);
```

Comment:
```suggestion
    pm_block(PM_MODE_EM2);
```

Reply:
This will only work when your PR #18760 is merged before. Alternatively, I could cherry-pick your commit https://github.com/RIOT-OS/RIOT/pull/18760/commits/6ec0135166544013116a139a4af99773c6f10e6f